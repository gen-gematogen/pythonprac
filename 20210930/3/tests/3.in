1,0,0
0,1,0
0,0,1
179,57,57
57,179,57
57,57,179
