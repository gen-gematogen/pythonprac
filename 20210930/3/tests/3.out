[179, 57, 57]
[57, 179, 57]
[57, 57, 179]
